rmw_connext_cpp/src/rmw_take.cpp

```diff
@@ -148,23 +148,22 @@ _take(
   DDS::InstanceHandle_t * sending_publication_handle,
   rmw_subscription_allocation_t * allocation)
 {
-  if (!subscription) {
-    RMW_SET_ERROR_MSG("subscription handle is null");
-    return RMW_RET_ERROR;
-  }
+  RMW_CHECK_FOR_NULL_WITH_MSG(
+    subscription, "subscription info is null",
```

Comment:
Why info instead of handle/pointer?
```suggestion
    subscription, "subscription handle is null",
```